pkg/plugin/backup_pvc_action_plugin.go

```diff
@@ -46,13 +46,27 @@ func (p *NewPVCBackupItemAction) Execute(item runtime.Unstructured, backup *vele
 		return item, nil, nil
 	}
 
+	blocked, crdName, err := utils.IsObjectBlocked(item)
+
+	if err != nil {
+		return nil, nil, errors.Wrap(err, "Failed during IsObjectBlocked check")
+	}
+
+	if blocked {
```

Comment:
Would it be easier if we only include pods as a resource on restore?  I think they are all independent.  Then we wouldn't get called for backup or delete (I think)

Reply:
Fixed.